packages/mermaid/src/docs/config/usage.md

```diff
@@ -183,9 +183,9 @@ Or with no config object, and a jQuery selection:
 mermaid.init(undefined, $('#someId .yetAnotherClass'));
 ```
 
-```warning
+::: warning
 This type of integration is deprecated. Instead the preferred way of handling more complex integration is to use the mermaidAPI instead.
-```
+:::
```

Comment:
You should keep code block in the source as that's valid markdown. `:::` is not. 
Then, during transformation, you can check for code blocks with `['warning','tip','note'].includes(language)` and do the conversion instead of using regex.

Reply:
it just fences the content in the transformed docs it shows with an Icon and i the webpage it has vitepress styles applied to it.